### PR TITLE
PLAT-99351: Fix minor docs issues

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fix minor moonstone docs issues

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fix incorrect information in moonstone component docs and correct typos.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-99351

### CODE REVIEW

File: Input/Input.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ const InputBase = kind({
 		 * The tooltip text to be displayed when the input is
 		 * [invalid]{@link moonstone/Input.InputBase.invalid}.
 		 *
-		 * If this value is *falsy*, the tooltip will not be shown.
+		 * If this value is *falsy*, the tooltip will be shown with default message.
 		 *
 		 * @type {String}
 		 * @default ''
```
Comment (jeonghee27) [inline Input/Input.js]:
```suggestion
		 * @default ''
```

Please add default value

**Review by jeonghee27 — APPROVED** [review]
LGTM

File: Input/Input.js (hunk 1 above)
Comment (juwonjeong) [reply Input/Input.js]: I added the default value.

**Review by jeonghee27 — APPROVED** [review]
LGTM

File: Input/Input.js [hunk 2]
```diff
@@ -112,10 +112,10 @@ const InputBase = kind({
 		 * The tooltip text to be displayed when the input is
 		 * [invalid]{@link moonstone/Input.InputBase.invalid}.
 		 *
-		 * If this value is *falsy*, the tooltip will not be shown.
+		 * If this value is *falsy*, the tooltip will be shown with default message.
```
Comment (MikyungKim) [inline Input/Input.js]:
```suggestion
		 * If this value is *falsy*, the tooltip will be shown with the default message.
```

Comment (juwonjeong) [reply Input/Input.js]: It's fixed.

File: Input/Input.js (hunk 1 above)
Comment (juwonjeong) [reply Input/Input.js]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**